lib/toggles/constant_lookup.rb

```diff
@@ -0,0 +1,47 @@
+class Feature::ConstantLookup
+  Error = Class.new(NameError) do
```

Comment:
why not just define this as, you know,

```
class Error < NameError do
   ...
end
```

Anonymous classes often feel like a code smell to me

Reply:
not an anonymous class, just a different way to declare a subclass.